src/process/executable_path.cr

```diff
@@ -90,6 +90,25 @@ end
       "/proc/self/exe"
     end
   end
+{% elsif flag?(:win32) %}
+  require "crystal/system/windows"
+  require "c/libloaderapi"
+
+  class Process
+    private def self.executable_path_impl
+      hMod = LibC.GetModuleHandleW(nil)
+      Crystal::System.retry_wstr_buffer do |buffer, small_buf|
+        len = LibC.GetModuleFileNameW(hMod, buffer, buffer.size)
```

Comment:
From the [docs](https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-getmodulefilenamew):

> hModule: A handle to the loaded module whose path is being requested. If this parameter is NULL, GetModuleFileName retrieves the path of the executable file of the current process.

So maybe we don't need to call `GetModuleHandleW`?

Reply:
Thanks for pointing. I'll remove `GetModuleHandleW`.

Reply:
I tested it on my local machine and pushed a new commit (568e1ad).